tools/topology/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "topology"
+version = "0.1.0"
+edition.workspace = true
+authors.workspace = true
+license.workspace = true
+
+[package.metadata.cargo-each]
+combinations = [
+    { tags = ["ci", "@agnostic"], include-rest = true }
+]
```

Comment:
We do and we should. Upholding quality standards is a must for me.

Reply:
let's reiterate why cargo-each was introduced .... it seems someone has missed the initial idea and what we agreed upon

Reply:
I'll ignore the passive-aggressive connotation and just say that UNTIL it goes away, we can't do much except not check, lint, run tests, etc. for said package.

If it were a random utility that I wrote purely for convenience, I'd still not agree but would be way more willing to compromise. Please, keep in mind that this will be fundamental part of the protocols, so not running tests is not desirable.

Reply:
Again, might not be possible because of how the DEX selection works right now. Moreover, it will be here to stay until I also make the generation of the sources too.

I know what the end goal is, but removing this is still out of reach for now.